src/role-access/containers/exclusion/describe-exclusion/describe-exclusion.component.spec.ts

```diff
@@ -1,15 +1,14 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { provideMockStore } from '@ngrx/store/testing';
-import { DescribeExclusionComponent } from './describe-exclusion.component';
+import { DescribeExclusionContainerComponent } from './describe-exclusion.component';
```

Comment:
I thought we wasn't using ContainerComponent anymore? The file is describe-exclusion.component so I expect the import should be the same.

Reply:
yes agreed with Paul

Reply:
Agree, done